src/web/routes/application/send-code/notify.js

```diff
@@ -0,0 +1,29 @@
+const { NotifyClient } = require('notifications-node-client')
+const uuid = require('uuid/v4')
+const config = require('../../../../config')
+const notifyClient = new NotifyClient(config.environment.NOTIFY_API_KEY)
```

Comment:
Ideally `config` would be passed in here rather than imported directly, although I realise this could complicate things. One approach would be:

```
// Initialise the client in behaviourForPost()
const behaviourForPost = (config) => {
  const notifyClient = new NotifyClient(config.environment.NOTIFY_API_KEY) 
  const sendConfirmationCodeWithNotify = sendConfirmationCode(notifyClient) 
  
  return (req, res, next) => {
    ...
    sendConfirmationCodeWithNotify(claim, channel, code)
    ...
  }
}

// Pass the client to sendConfirmationCode()
const sendConfirmationCode = (client) => (claim, channel, code) => { ... }
```

Reply:
Unfortunately this doesn't seem to be that simple due to changes in register-form-routes. Happy to pair with it in the next PR as doing it in this PR would require an additional file change.

Reply:
OK, for the sake of 2fa this is fine for now. I would really like to come back to this so will have a think about it. I think maybe the solution would be to pass `config, steps, step` to all middleware in `registerFormRoutes` as generally most steps end up needing these parameters at some point.